src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
+@import "./utils/variables";
 @import "blocks/page";
 @import "blocks/header";
 @import "blocks/catalog";
```

Comment:
I think you need to change the path to the block folder

Reply:
Path was renewed